internal/pkg/cli/deploy.go

```diff
@@ -186,45 +196,196 @@ func (o *deployOpts) maybeInitWkld() error {
 	}
 
 	// Workload is already initialized. Return early.
-	if slices.Contains(wlNames, o.name) {
+	if slices.Contains(wlNames, name) {
 		return nil
 	}
 
 	// Get workload type and confirm readable manifest.
-	mf, err := o.ws.ReadWorkloadManifest(o.name)
+	mf, err := o.ws.ReadWorkloadManifest(name)
 	if err != nil {
-		return fmt.Errorf("read manifest for workload %s: %w", o.name, err)
+		return fmt.Errorf("read manifest for workload %s: %w", name, err)
 	}
 	workloadType, err := mf.WorkloadType()
 	if err != nil {
-		return fmt.Errorf("get workload type from manifest for workload %s: %w", o.name, err)
+		return fmt.Errorf("get workload type from manifest for workload %s: %w", name, err)
 	}
 
 	if !slices.Contains(manifestinfo.WorkloadTypes(), workloadType) {
-		return fmt.Errorf("unrecognized workload type %q in manifest for workload %s", workloadType, o.name)
+		return fmt.Errorf("unrecognized workload type %q in manifest for workload %s", workloadType, name)
 	}
 
 	if o.yesInitWkld == nil {
-		confirmInitWkld, err := o.prompt.Confirm(fmt.Sprintf("Found manifest for uninitialized %s %q. Initialize it?", workloadType, o.name), "This workload will be initialized, then deployed.", prompt.WithFinalMessage(fmt.Sprintf("Initialize %s:", workloadType)))
+		confirmInitWkld, err := o.prompt.Confirm(fmt.Sprintf("Found manifest for uninitialized %s %q. Initialize it?", workloadType, name), "This workload will be initialized, then deployed.", prompt.WithFinalMessage(fmt.Sprintf("Initialize %s:", workloadType)))
 		if err != nil {
 			return fmt.Errorf("confirm initialize workload: %w", err)
 		}
 		o.yesInitWkld = aws.Bool(confirmInitWkld)
 	}
 
 	if !aws.BoolValue(o.yesInitWkld) {
-		return fmt.Errorf("workload %s is uninitialized but --%s=false was specified", o.name, yesInitWorkloadFlag)
+		return fmt.Errorf("workload %s is uninitialized but --%s=false was specified", name, yesInitWorkloadFlag)
 	}
 
 	wkldAdder := o.newWorkloadAdder()
-	if err = wkldAdder.AddWorkloadToApp(o.appName, o.name, workloadType); err != nil {
+	if err = wkldAdder.AddWorkloadToApp(o.appName, name, workloadType); err != nil {
 		return fmt.Errorf("add workload to app: %w", err)
 	}
 	return nil
 }
 
+func parseDeploymentOrderTags(namesWithOptionalOrder []string) (map[string]int, error) {
+	prioritiesMap := make(map[string]int)
```

Comment:
Why do we use "priorities" instead of "orders" here?

Reply:
this is a first step to parallelizing deployments--the idea is to create multiple groups which may be deployed in parallel, and then deploy those groups in a prescribed order. I wanted to use a different word to distinguish this, which is a precursor data structure, from the final.